prepareNewVersion.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+if [[ -z "$1" ]]; then
+    echo First argument must be new version to set
+    exit 1
+fi
+
+version=$1
+
+# prepare release with maven
+./mvnw -f parent-pom.xml versions:set -DnewVersion=$version -DgenerateBackupPoms=false
+
+# update version in Driver code
+version_without_snapshot=${version%-*}
+file_with_version=src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java
+tmp_file_with_version=${file_with_version}.tmp
+sed -E "s/( implementVersion = )(.+)(;)/\1\"${version_without_snapshot}\"\3/" src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java > $tmp_file_with_version
+mv $tmp_file_with_version $file_with_version
```

Comment:
Why not use `sed -i` instead?

Reply:
I was struggling with using`-i` with `-E` - sed was constantly creating file with suffix
Creating a temporary file was an easy solution there